src/Mod/Sketcher/Gui/CommandConstraints.cpp

```diff
@@ -2073,40 +2065,64 @@ class DrawSketchHandlerDimension : public DrawSketchHandler
                 Base::Vector3d di = ct - pnt;
                 ActDist = std::abs(di.Length() - circle->getRadius());
             }
+            else if (isArcOfCircle(*geom)) {
+                auto arc = static_cast<const Part::GeomArcOfCircle*>(geom);
+                Base::Vector3d ct = arc->getCenter();
+                Base::Vector3d di = ct - pnt;
+                ActDist = std::abs(di.Length() - arc->getRadius());
+            }
 
             Gui::cmdAppObjectArgs(Obj, "addConstraint(Sketcher.Constraint('Distance',%d,%d,%d,%f)) ",
                 GeoId1, static_cast<int>(PosId1), GeoId2, ActDist);
         }
-        else if (PosId1 == Sketcher::PointPos::none && PosId2 == Sketcher::PointPos::none) { // Circle - line, circle - circle cases
+        else if (PosId1 == Sketcher::PointPos::none
+                 && PosId2 == Sketcher::PointPos::none) {  // Circle/arc - line, circle/arc -
+                                                           // circle/arc cases
```

Comment:
clang is forcing this weird syntax because the line was becoming too long. So better put the comment on the bottom line instead
```
else if (PosId1 == Sketcher::PointPos::none && PosId2 == Sketcher::PointPos::none) { 
// Circle/arc - line, circle/arc - circle/arc cases
```